packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -203,6 +203,68 @@ class DateTimePicker extends React.Component {
     }
   }
 
+  getMetadata(momentDateTime) {
+    const isCompleteDateTime = DateTimeUtils.isValidDateTime(this.dateValue, this.timeValue, this.state.dateFormat, this.props.showSeconds);
+    let value = '';
```

Comment:
Let's be explicit here. I'd rename it to something like - `dateTimeInputValue` / `inputValue`.

Reply:
Addressed here : https://github.com/cerner/terra-framework/pull/1045/commits/1e8dc74b082dc0de2c1002be51b425e1d49f9784